clients/src/main/java/org/apache/kafka/common/metrics/KafkaMetric.java

```diff
@@ -78,6 +109,11 @@ public Measurable measurable() {
         }
     }
 
+    /**
+     * Set the metric config.
+     * This is supposed to be used by server only, DO NOT use this for your metrics.
```

Comment:
`This is supposed to be used by server only` this description is good enough to me